java/kafka/deployment-ssl.yaml

```diff
@@ -82,21 +82,31 @@ spec:
         - name: java-kafka-streams
           image: quay.io/strimzi-examples/java-kafka-streams:latest
           env:
-            - name: CA_CRT
+            - name: STRIMZI_SOURCE_TOPIC
+              value: my-topic
+            - name: STRIMZI_TARGET_TOPIC
+              value: my-topic-reversed
+            - name: STRIMZI_LOG_LEVEL
+              value: "INFO"
+            - name: KAFKA_BOOTSTRAP_SERVERS
+              value: my-cluster-kafka-bootstrap:9093
+            - name: KAFKA_APPLICATION_ID
+              value: java-kafka-streams
+            - name: KAFKA_DEFAULT_COMMIT_INTERVAL_MS
+              value: "5000"
+            - name: KAFKA_DEFAULT_KEY_SERDE
+              value: "org.apache.kafka.common.serialization.Serdes$StringSerde"
+            - name: KAFKA_DEFAULT_VALUE_SERDE
+              value: "org.apache.kafka.common.serialization.Serdes$StringSerde"
+            - name: KAFKA_SSL_TRUSTSTORE_CERTIFICATES
               valueFrom:
                 secretKeyRef:
                   name: my-cluster-cluster-ca-cert
                   key: ca.crt
-            - name: BOOTSTRAP_SERVERS
-              value: my-cluster-kafka-bootstrap:9093
-            - name: APPLICATION_ID
-              value: java-kafka-streams
-            - name: SOURCE_TOPIC
-              value: my-topic
-            - name: TARGET_TOPIC
-              value: my-topic-reversed
-            - name: LOG_LEVEL
-              value: "INFO"
+            - name: KAFKA_SECURITY_PROTOCOL
+              value: SSL
+            - name: KAFKA_SSL_TRUSTSTORE_TYPE
+              value: PEM
```

Comment:
Maybe you can switch these to have it somehow grouped? `KAFKA_SSL_TRUSTSTORE_TYPE` belongs to `KAFKA_SSL_TRUSTSTORE_CERTIFICATES`.

Reply:
Grouped now